src/styles/card.scss

```diff
@@ -0,0 +1,44 @@
+@import './variables';
```

Comment:
Ensure that the `variables` file contains definitions for `$card-width`, `$primary-font`, and `$blue-color`. Without these, the SCSS compilation will fail.